accounts-db/src/read_only_accounts_cache.rs

```diff
@@ -86,9 +86,10 @@ pub(crate) struct ReadOnlyAccountsCache {
     ///
     /// NOTE: This field must be above `evictor` to ensure it is dropped before `evictor`.
     evict_sender: crossbeam_channel::Sender<()>,
-    /// To the evictor goes the spoiled [sic]
-    ///
-    /// Evict from the cache in the background.
+    // To the evictor goes the spoiled [sic]
```

Comment:
nit: If there's three slashes, Cargo doc adds it to the documentation for the field.

Reply:
Yes, but now that field doesn't exist sometimes due to the cfg, and if there's a /// comment without something after it, it's a compile error.

Reply:
Oh right, thanks for pointing that out.